Assets/Scripts/Networking/EOSTransportManager.cs

```diff
@@ -1112,6 +1112,15 @@ public bool TryReceivePacket(out ProductUserId remoteUserId, out string socketNa
             };
             P2PHandle.GetNextReceivedPacketSize(ref getNextReceivedPacketSizeOptions, out uint nextPacketSizeBytes);
 
+            if (nextPacketSizeBytes == 0)
+            {
+                remoteUserId = null;
+                socketName = null;
+                channel = 0;
+                packet = null;
+                return false;
+            }
```

Comment:
Can't comment on other sections of code, but this is adjacent. Would it not make sense to move lines 1101-1106 to be below this section of code? Since `receivePackatOptions` isn't used until later in the function?

Reply:
Yes that would make sense to do. I somehow didn't see this before I merged this PR in. But that was how the code was already laid out. I agree with you though. I always put the options struct directly above the EOS SDK call that uses it.